p2p/sync.py

```diff
@@ -39,6 +40,14 @@ def __init__(self,
         self.peer_pool = peer_pool
 
     async def _run(self) -> None:
+        try:
+            await self._sync()
+        except OperationCancelled:
```

Comment:
You shouldn't catch `OperationCancelled` here as `p2p.BaseService` will do that and log it